entity-management-web/src/main/java/eu/europeana/entitymanagement/web/EMController.java

```diff
@@ -721,8 +723,31 @@ private ResponseEntity<String> createResponseRetrieve(
       String languages,
       String contentType)
       throws EuropeanaApiException {
-    EntityRecord entityRecord =
-        entityRecordService.retrieveEntityRecord(type, identifier.toLowerCase(), true);
+    
+    EntityRecord entityRecord = null;
+    try {
+      entityRecord = 
+          entityRecordService.retrieveEntityRecord(type, identifier.toLowerCase(), true);
+    }
+    catch (EntityNotFoundException ex) {
```

Comment:
this is part of the business logic, must be moved to a service method. Something like findRedirection